tests/test_retokenize.py

```diff
@@ -337,13 +337,13 @@ def test_bytebpe(self):
         ]
         self.token_index_tgt = [
             [[0], [1], [2], [3], [4, 5], [6], [7]],
-            [[0], [1], [2], [3, 4], [5], [6, 7], [8], [9, 10, 11]],
-            [[0], [1, 2, 3], [4], [5], [6], [7], [8], [9, 10, 11], [12], [13], [14, 15]],
+            [[0], [1], [2], [3, 4], [5, 6], [7], [8], [9, 10, 11]],
+            [[0, 1], [2, 3], [4], [5], [6], [7], [8], [9, 10, 11], [12], [13], [14, 15]],
```

Comment:
I wasn't computing this manually. I just printed out the output and skim over it.
I know this isn't how test cases are suppose to be made.
Me trying to save 5 minutes in the wrong place has wasted everyone else hours, there can't be enough apology for this.